kura/pom.xml

```diff
@@ -204,14 +205,15 @@
                     -Dkura.home=${project.basedir}/target/kura
                     -Dkura.plugins=${project.basedir}/target/kura/plugins
                     -Dkura.packages=${project.basedir}/target/kura/data/packages
-                    -Dkura.data=${project.basedir}/target/kura
+                    -Dkura.data=${project.basedir}/target/kura/data
```

Comment:
I think java 8 profile can be removed here, and the Java 17 settings can be set as default configuration in the plugin